h/views/groups.py

```diff
@@ -87,6 +90,15 @@ def api_config(route_name: str, method: str, **params):
                     "readGroupMembers": api_config(
                         "api.group_members", "GET", pubid=group.pubid
                     ),
+                    "editGroupMember": api_config(
+                        "api.group_member", "PATCH", pubid=group.pubid, userid=":userid"
+                    ),
+                    "removeGroupMember": api_config(
+                        "api.group_member",
+                        "DELETE",
+                        pubid=group.pubid,
+                        userid=":userid",
+                    ),
```

Comment:
We are gradually starting to duplicate parts of the API index at https://hypothes.is/api/. Down the line I wonder if it might make sense to have the frontend use that in some way. There are differences though - some of the parameters are fixed, there are additional auth headers and only a subset of the API is needed.